code/modules/mob/living/carbon/carbon.dm

```diff
@@ -767,28 +767,33 @@
 	else
 		hud_used.healths.icon_state = "health6"
 
-/mob/living/carbon/update_stamina_hud(shown_stamina_amount)
+/mob/living/carbon/update_stamina_hud(shown_stamina_loss)
```

Comment:
Can you update the base of this and anything else that might override to use this variable name? Otherwise named params are harder

Reply:
![image](https://user-images.githubusercontent.com/55666666/205420450-5ba5e2a1-b415-4506-9108-b7c5ffa176c6.png)

That var wasn't used anywhere else.
![image](https://user-images.githubusercontent.com/55666666/205420478-39db2753-068b-4aa0-86d6-984bdd462697.png)

Reply:
This was the only context that var was ever called or used. Its probably a holdover from halloss and perhaps warrants removal of the functionality entirety.